src/ORM/Rule/ExistsIn.php

```diff
@@ -98,6 +109,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        if ($this->_options['passingOnPartialNulls'] === true
```

Comment:
I changed it. However I don't understand why it is wrong to be explicit about types `in general`.